tpm2/include/kmyth.h

```diff
@@ -35,11 +35,12 @@
  *
  * @param[in]  oa_bytes_len      number of bytes in owner_auth_passwd
  *
- * @param[in]  pcrs_string       String indicating which PCRs, if any, to apply
+ * @param[in]  pcrs              Array containing PCRs values, if any, to apply
```

Comment:
This is nit-picky, but I think this comment is potentially a little confusing. The array contains 'PCR selections' or 'PCR indices'. When I hear 'PCR values', I think of the contents of PCR registers, but perhaps the context makes it obvious that we are identifying PCRs not reading from them.